lib/bootsnap/load_path_cache/cache.rb

```diff
@@ -24,8 +24,16 @@ def load_dir(dir)
         @mutex.synchronize { @dirs[dir] }
       end
 
+      TRUFFLERUBY_LIB_DIR_PREFIX = if RUBY_ENGINE == "truffleruby"
+        "#{File.join(RbConfig::CONFIG['libdir'], 'truffle')}#{File::SEPARATOR}"
```

Comment:
Minor, but you can use`File.join(RbConfig::CONFIG['libdir'], 'truffle', '')`. Ruby will append the trailing separator in that case and you can avoid the string interpolation.

Reply:
i started with that originally but wasn't confident that it wouldn't get normalized out in some version.